pkg/framework/plugins/defaultevictor/defaultevictor.go

```diff
@@ -154,6 +146,21 @@ func (d *DefaultEvictor) Name() string {
 	return PluginName
 }
 
+func (d *DefaultEvictor) PreEvictionFilter(pod *v1.Pod) (bool, error) {
```

Comment:
I still prefer to return only bool in the same fashion as `Filter` and log any error the same way. Later on we can decide whether it's more preferable to return a status instead of a bool.